dev/io.openliberty.checkpoint_fat_transaction/fat/src/io/openliberty/checkpoint/fat/MockServletTest.java

```diff
@@ -46,7 +46,7 @@
 import io.openliberty.checkpoint.spi.CheckpointPhase;
 
 @RunWith(FATRunner.class)
-@SkipIfCheckpointNotSupported
```

Comment:
@tjwatson: Noticed I declared a skip annotation on this mock checkpoint test. That was unnecessary, but it got me thinking: Given the intent to automagically run all checkpoint FATs on specialized build definitions, should we annotate mock tests as well to ensure these builds provide the entire result set and to potentially reduce the overhead of these FATs on other specialized build definitions.  The overhead is minor, but why would we care to exercise mock checkpoint FATs on windows, etc.?

Reply:
I don't think we should annotate mock tests to not run on non-Linux.  Mock tests help enable developers running on non-linux to test out their code locally.  We should use the `alwaysRun` attribute on them to mark them as `CheckpointTest` though I think.